wp-config.php.j2

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
where is it used?

Reply:
File to connect php with mysql database.